### DISCUSSION TITLE
Why doesn't my code work to open wezterm current dir via cmd?

### DISCUSSION THREAD

**Discussion by xtheeq** [moudey/Shell#492] [Q&A]
**Config path:** directory/imports/terminal.nss

I included 
`
item(title="Wezterm" tip=tip_run_admin admin=has_admin cmd='wezterm.exe')
`
which opens wezterm

**but**

when i want to open current dir like it does for other terminals it does not work
this was my code:
`
item(title="Wezterm" tip=tip_run_admin admin=has_admin cmd='wezterm start --cwd "@sel.dir"')
`

wezterm [reference](https://wezfurlong.org/wezterm/config/launch.html#specifying-the-current-working-directory)

**Comment by moudey** [comment]
```js
 item(title="Wezterm" tip=tip_run_admin admin=has_admin cmd='wezterm.exe' args='start --cwd "@sel.dir"')
```

**Reply by xtheeq** [reply]
tysm

**Reply by xtheeq** [reply]
btw can u say how to get the icon image?

**Reply by moudey** [reply]
```js
item(title="Wezterm" admin=has_admin 
image
cmd='wezterm.exe' args='start --cwd "@sel.dir"')
```
or
```js
item(title="Wezterm" admin=has_admin 
image = \ue249
cmd='wezterm.exe' args='start --cwd "@sel.dir"')
```

**Reply by xtheeq** [reply]
> ```js
> item(title="Wezterm" admin=has_admin 
> image
> cmd='wezterm.exe' args='start --cwd "@sel.dir"')
> ```

I tried this but it didn't work, no way around to get the icon of wezterm itself like other terminal?

**Reply by xtheeq** [reply]
why open in terminal comes when there is no calling for it explicitly?

**Reply by moudey** [reply]
```js
image = "wezterm-gui.exe"
```

**Reply by xtheeq** [reply]
> ```js
> image = "wezterm-gui.exe"
> ```

it didn't work

**Reply by KellaAkella** [reply]
> > ```js
> > image = "wezterm-gui.exe"
> > ```
> 
> it didn't work

item(title="Wezterm" tip=tip_run_admin admin=has_admin image = 'path to your wezterm-gui.exe' cmd='wezterm.exe' args='start --cwd "@sel.dir"') worked for me. Instead of double quotes, use single(apostrophe).

and change cmd='wezterm.exe' to cmd='wezterm-gui.exe' so it will launch wezterm directly instead of launching Windows Terminal which launces wezterm.
